includes/class-wc-stripe-helper.php

```diff
@@ -376,7 +376,7 @@ public static function get_settings( $method = null, $setting = null ) {
 	 * @return bool
 	 */
 	public static function is_pre_orders_exists() {
-		return class_exists( 'WC_Pre_Orders_Order' );
+		return class_exists( 'WC_Pre_Orders' );
```

Comment:
This change is needed because WC_Pre_Orders_Order does not exist until after plugins_loaded. Since we are doing this check in the init during plugins_loaded we need to check for the `WC_Pre_Orders` class which will always be present when the plugin is active